[2024-05-05T10:22:42.272+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-05T10:22:42.356+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_complex.update_entry manual__2024-05-05T10:22:28.612296+00:00 [queued]>
[2024-05-05T10:22:42.381+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_complex.update_entry manual__2024-05-05T10:22:28.612296+00:00 [queued]>
[2024-05-05T10:22:42.381+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-05T10:22:42.411+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): update_entry> on 2024-05-05 10:22:28.612296+00:00
[2024-05-05T10:22:42.437+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'example_complex', 'update_entry', 'manual__2024-05-05T10:22:28.612296+00:00', '--job-id', '7', '--raw', '--subdir', '/home/***/.local/lib/python3.12/site-packages/***/example_dags/example_complex.py', '--cfg-path', '/tmp/tmpdyttqm88']
[2024-05-05T10:22:42.450+0000] {standard_task_runner.py:91} INFO - Job 7: Subtask update_entry
[2024-05-05T10:22:42.450+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=2014) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-05T10:22:42.454+0000] {standard_task_runner.py:63} INFO - Started process 2028 to run task
[2024-05-05T10:22:42.592+0000] {task_command.py:426} INFO - Running <TaskInstance: example_complex.update_entry manual__2024-05-05T10:22:28.612296+00:00 [running]> on host b2b5d11d7cf5
[2024-05-05T10:22:43.118+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_complex' AIRFLOW_CTX_TASK_ID='update_entry' AIRFLOW_CTX_EXECUTION_DATE='2024-05-05T10:22:28.612296+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-05T10:22:28.612296+00:00'
[2024-05-05T10:22:43.119+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-05T10:22:43.121+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-05T10:22:43.121+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo update_entry']
[2024-05-05T10:22:43.184+0000] {subprocess.py:86} INFO - Output:
[2024-05-05T10:22:43.197+0000] {subprocess.py:93} INFO - update_entry
[2024-05-05T10:22:43.198+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-05T10:22:43.198+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-05T10:22:43.286+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=example_complex, task_id=update_entry, execution_date=20240505T102228, start_date=20240505T102242, end_date=20240505T102243
[2024-05-05T10:22:43.409+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-05T10:22:43.451+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-05T10:22:43.484+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
